modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalAddObjectDefinition.tsx

```diff
@@ -199,34 +205,63 @@ export function ModalAddObjectDefinition({
 						/>
 
 						{Liferay.FeatureFlags['LPS-135430'] && (
-							<div className="lfr__object-web-modal-add-object-definition-storage-type">
-								<SingleSelect<LabelValueObject>
-									items={
-										objectDefinitionStorageTypesSortedByLabel
-									}
-									label={Liferay.Language.get('storage-type')}
-									onSelectionChange={(value) => {
-										setValues({
-											...values,
-											storageType: value as string,
-										});
-									}}
-									selectedKey={values.storageType}
-									tooltip={Liferay.Language.get(
-										'object-definition-storage-type-tooltip'
-									)}
-								/>
-
-								<div className="lfr__object-web-modal-add-object-definition-storage-type-beta">
-									<FeatureIndicator
-										interactive
-										learnResourceContext={
-											learnResourceContext
+							<>
+								<div className="lfr__object-web-modal-add-object-definition-storage-type">
+									<SingleSelect<LabelValueObject>
+										items={
+											objectDefinitionStorageTypesSortedByLabel
 										}
-										type="beta"
+										label={Liferay.Language.get(
+											'storage-type'
+										)}
+										onSelectionChange={(value) => {
+											setValues({
+												...values,
+												storageType: value as string,
+											});
+											setShowProxyWarning(true);
```

Comment:
Hey @DavysonMelo, if you are setting the state without any custom logic, do you really need of this state? 

What would be the harm in using only `values.storage Type === 'proxy'`, for example, to condition the rendering of the alert?